qutebrowser/browser/commands.py

```diff
@@ -985,6 +987,37 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         url = objreg.get('quickmark-manager').get(name)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', name='select-follow',
+                       scope='window')
+    def select_follow(self, tab=False):
+        """Follow the selected text.
+
+        Args:
+            tab: Load the selected link in a new tab.
+        """
+        widget = self._current_widget()
+        page = widget.page()
+        if QWebSettings.globalSettings().testAttribute(
+                QWebSettings.JavascriptEnabled):
+            if tab:
+                page.open_target = usertypes.ClickTarget.tab
+            page.currentFrame().evaluateJavaScript(
+                'window.getSelection().anchorNode.parentNode.click()')
+        else:
+            try:
+                selected_element = xml.etree.ElementTree.fromstring(
+                    '<html>' + widget.selectedHtml() + '</html>').find('a')
+            except xml.etree.ElementTree.ParseError:
+                raise cmdexc.CommandError('Parse error!')
```

Comment:
This might be a confusing error message - maybe `Could not parse selected element!`?